src/Token.php

```diff
@@ -65,21 +65,21 @@ public static function getTokenKindNameFromValue($kindName) {
     function jsonSerialize() {
         $kindName = $this->getTokenKindNameFromValue($this->kind);
 
-        if (!isset($GLOBALS["SHORT_TOKEN_SERIALIZE"])) {
-            $GLOBALS["SHORT_TOKEN_SERIALIZE"] = false;
+        if (!isset($GLOBALS['SHORT_TOKEN_SERIALIZE'])) {
```

Comment:
Yeah there isn't. I was still under the idea that single quotes are faster. But as they aren't this can be closed IMHO